### PR TITLE
fix(search-service): export datasource name as expected by cli in search service

### LINKED ISSUE #2136: Search-service not starting by generating with cli
**Describe the bug**
 search-service are not starting , because DATASOURCE_NAME variable is not correctly exported.
 currently in Search service its exported as `SearchServiceBindings.DATASOURCE_NAME`.

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
export datasource name as expected by cli in search service

GH-2136

## Description
export datasource name as expected by cli in search service

Fixes #2136 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/search-service/src/keys.ts [hunk 1]
```diff
@@ -30,3 +30,4 @@ export namespace SearchServiceBindings {
 }
 
 export const IGNORED_COLUMN = '-';
+export const DATASOURCE_NAME = 'SearchServiceDb';
```
Comment (samarpan-b) [inline services/search-service/src/keys.ts:33]: Where is it used ?

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: services/search-service/src/keys.ts (hunk 1 above)
Comment (Surbhi-sharma1) [reply services/search-service/src/keys.ts:33]: Exported datasourceName as required by the CLI to resolve the 'datasource name not exported' error while using search service. currently its exported as SearchServiceBindings.DATASOURCE_NAME .

Comment (yeshamavani) [reply services/search-service/src/keys.ts:33]: the datasource key is required while we generate search service through cli 
for all the services it is directly exported but in search search-service it is exported via a 'SearchServiceBindings.DATASOURCE_NAME'

**Comment by yeshamavani** [comment]
@Surbhi-sharma1 revert the package-lock file changes

**Comment by Surbhi-sharma1** [comment]
> @Surbhi-sharma1 revert the package-lock file changes

Done

**Pull request merged**